packages/ckeditor5-image/src/imageupload/imageuploadediting.ts

```diff
@@ -135,6 +137,14 @@ export default class ImageUploadEditing extends Plugin {
 
 				editor.execute( 'uploadImage', { file: images } );
 			} );
+
+			const uploadImageCommand = editor.commands.get( 'uploadImage' )!;
+			const t = this.editor.locale.t;
+
+			if ( !uploadImageCommand.isAccessAlowed ) {
+				window.alert( t( 'No permission to upload from computer. Try to use file ' + // eslint-disable-line no-alert
```

Comment:
Shouldn't we use `global` from `ckeditor5/src/utils.js`? e.g. `global.window.alert(...` :thinking:

Reply:
We finally decided to go with `notification` option: https://github.com/ckeditor/ckeditor5/pull/16814#discussion_r1696590964